tasks/insights-client.yml

```diff
@@ -36,6 +36,29 @@
     insertafter: "#auto_update"
     line: auto_update={{ rhc_insights.autoupdate | d(true) | bool }}
 
+- name: Configure ansible host
+  when:
+    - not rhc_insights.ansible_host is none
+    - rhc_insights.ansible_host != omit
+  block:
+    - name: Configure ansible host in config
+      lineinfile:
+        path: "{{ __rhc_insights_conf }}"
+        regexp: "^ansible_host"
```

Comment:
put `=` in the regexp right after the key name, e.g.:
```        regexp: "^ansible_host="
```
this way, if there would ever be yet another option named `ansible_host<something>`, it would not be affected by this block.